data/successors/successor 1 introduction.txt

```diff
@@ -0,0 +1,1020 @@
+			`	The Kijra-pet trills as it eats out of your hand; it tucks its wings behind its body like a dragonfly and unfurls two long feather-like antennae from its head, which gently brush at your skin and clothing. Soon, it hops on your arm while sniffing at your pockets, probing deeper into the one where you kept the food and trilling softly at you. You begin the walk back to your ship, slowly and gently so as not to frighten the animal.`
+			`	"'Tis no surprise," Sieasej says, "that 'twere such valued pets if their demeanor is so docile and agreeable." They tuck their triangulation device away in a pocket and slither with you as you approach your ship.`
+				goto ambush
+			label tranq-friends
+			`	While the animal is distracted, you take aim and fire a shot from your <tranq h2h> at it. It briefly jumps to escape but soon plummets to the ground in a puff of leaves and dusty feathers.`
+				goto bring-tranq
+			label net-friends-alarmed
+			`	While the animal is distracted, you toss a net over it and it falls to the ground, where it shrieks for a duration before going quiet. The shriek is soon echoed by similar ones from other animals in the area, and once those go quiet as well, your ears are pierced by a loud, warbling wail in the distance.`
+				goto escape
+			label bring-tranq
+			`	You scoop up the tranquilized animal, careful not to damage its extremities, and begin the walk back to your ship.`
+				goto ambush
+			label escape
+			choice
+				`	(Grab it and run!)`
+			`	You scoop up the immobilized animal and begin to run back to your ship as the wailing behind you increases in pitch and intensity. As you and Sieasej dash through the foliage, the sunlight overhead is suddenly darkened by the shadow of something flying overhead.`
+			choice
+				`	(Keep running.)`
+				`	(Try to shoot at whatever is above you.)`
+					goto shoot-ambush
+			`	Suddenly, the shadow disappears and the wailing stops. You keep running towards your ship, but can't shake the feeling that you are being watched.`
+				goto ambush
+			label shoot-ambush
+			`	You draw your sidearm and look to see what's above you, but the shadow is suddenly gone and you are left staring up into the sunlight and leaves. The wailing has stopped as well; indeed, the whole area remains eerily quiet as you continue to run towards your ship.`
+			label ambush
+			`	When you arrive, however, you find that blocking your entry to the <ship> are two large creatures. Their glassy skin resembles that of the smaller creature you captured, but they are each several meters in size. Their feelers, rather than soft and downy, are whiplike and serrated and tipped with vicious-looking barbs that they thrust towards you and Sieasej upon seeing you. Their many eyes flick back and forth between you and the animal that you captured, and the two let out a loud wail in unison. Waves of red and blue run down Sieasej's body and you have to imagine it's an expression of fear.`
```

Comment:
```suggestion
			`	When you arrive, however, you find that blocking your entry to the <ship> are two large creatures. Their glassy skin resembles that of the smaller creature you captured, but they are each several meters in size. Their feelers, rather than soft and downy, are whiplike and serrated and tipped with vicious-looking barbs that they thrust towards you and Sieasej upon seeing you. Their many eyes flick back and forth between you and the animal that you captured, and the two let out a loud wail in unison. Waves of red and blue run down Sieasej's body, apparently an expression of fear.`
```